skopt/tests/test_common.py

```diff
@@ -244,7 +244,7 @@ def test_init_points_and_models(n_random_starts, optimizer_func):
 def test_init_vals(n_random_starts, optimizer_func):
     space = [(-5.0, 10.0), (0.0, 15.0)]
     x0 = [[1, 2], [3, 4], [5, 6]]
-    n_calls = 10
+    n_calls = len(x0) + n_random_starts
```

Comment:
I would suggest this to reduce this rather to check the case where this exceeds len(x0) + n_random_starts